### DISCUSSION TITLE
multi view 'Context' Not very easy to use

### DISCUSSION THREAD

**Discussion by a44281071** [Caliburn-Micro/Caliburn.Micro#814] [Ideas]
Param is type Object but it is only used string name path.

Affected the naming, 

``` csharp

// model
    class Student { }
    class Teacher { }
// view_model
    class StudentViewModel { }
// view
    class StudentView { }
// view with context
namespace Student
{
    class Details { }
}
namespace Teacher
{
    class Details { }
}

```

The possible problem is
- Lost tail View for view with context.
- Class namespace same with model class.
- Same class name.

Change context naming convention, for example:
```
Student
StudentViewModel  // name + ViewModel
StudentView             // name + View
StudentDetailsVIew  // name + context + View
```

**Comment by vb2ae** [comment]
What ViewModel would you expect StudentDetailsView to use?

**Reply by a44281071** [reply]
StudentDetailsView use StudentViewModel
